examples/distro/build.gradle

```diff
@@ -30,6 +30,7 @@ subprojects {
 
   repositories {
     mavenCentral()
+    mavenLocal()
```

Comment:
We publishToMavenLocal in the workflow but pretty sure I needed to add this here to get that to kick in